packages/esm-service-queues-app/src/patient-queue-metrics/clinic-metrics.component.tsx

```diff
@@ -57,7 +57,7 @@ function ClinicMetrics() {
         />
         <MetricsCard
           label={t('patients', 'Patients')}
-          value={initialSelectedItem ? totalCount ?? '--' : serviceCount}
+          value={initialSelectedItem ? totalCount || '--' : serviceCount}
```

Comment:
Used or Operator here because useOpenMrsFetchAll returns NaN when there is no total count and nullish collasing operator doesn't recognize NaN as faulty value. So we see for a Brief moment NaN being displayed in the metrics card .
We can overcome this using OR operator.

Reply:
hm... but if `totalCount` is 0, we will just get `'--'`. I think we'll have to manually check for NaN (or `isLoading` from `useOpenMrsFetchAll`)

Reply:
yeah, correct . I will make the corrections. Thanks.